src/templates/new_relations.tsv

```diff
@@ -1,2 +1,16 @@
 relation id (required)	type	label (required)	definition (required)	parent relation	creator ORCID (required)	Date created (required)	relation characteristics	domain	range	inverse property	example of use	synonyms	curator/usage notes
 ID	TYPE	AL rdfs:label@en	A IAO:0000115	SP %	AI dc:contributor	AT dc:created^^xsd:dateTime	CHARACTERISTIC SPLIT=|	DOMAIN	RANGE	IP %	A rdfs:comment	A oboInOwl:hasExactSynonym SPLIT=|	A IAO:0000116
+20001 TYPE  isAnatomicalEntityFor A relation describing an anatomical featureof the referenced object with a material entity   AI dc:contributor0000-0002-6571-0033 AT dc:created^^xsd:16.03.2022 
+20002 TYPE  isBrainConnectivityFor  The subject describes the estimated relation between brain regions in a given condition (object). 
+20003 TYPE  isCellularProcessFor  The subject describes any process that is carried out at the cellular level, but not necessarily restricted to a single cell for the object.  
+20004 TYPE  isClassificationFor The subject is a subcategory of the object with certain criteria. 0000-0002-6571-0033 16.03.2022
+20005 TYPE  isDiagnosisFor  The subject is a technique carried out in a clinical assessment for evaluation of a condition (object). 

```

Comment:
every isXFor should have a corresponding `has X` relation. So you should also add another `has diagnosis` relation here.